src/components/Breadcrumbs.js

```diff
@@ -22,10 +22,17 @@ export default class Breadcrumbs extends Component {
       const paths = splat.split('/');
       links = _.map(paths, (path, i) => {
         const before = (i == 0) ? '' : paths.slice(0, i).join('/') + '/';
-        return {
-          href: `${base}/${before}${path}`,
-          label: path
-        };
+        if (type == "data files") {
```

Comment:
I'm suggesting to fix it since we are proposing sub-directory support now.
@benbalter What do you think?